airflow/utils/dag_cycle_tester.py

```diff
@@ -25,6 +25,21 @@
 
 
 def test_cycle(dag):
+    """
+    A wrapper function of `check_cycle` for backward compatibility purpose.
+    New code should use `check_cycle` instead since this function name `test_cycle` starts with 'test_' and
+    will be considered as a unit test by pytest, resulting in failure.
+    """
+    from warnings import warn
```

Comment:
Why does this import need to be local?

Reply:
We discourage the usage of this function and warn that it is deprecated, as a result, its usage can be predicted to be rare. This 'warn' is only used here so doesn't need to be imported at the module level.